odoo/fields.py

```diff
@@ -1002,7 +1002,7 @@ def update_db_index(self, model, column):
                 sql.create_index(model._cr, indexname, model._table, ['"%s"' % self.name])
             except psycopg2.OperationalError:
                 _schema.error("Unable to add index for %s", self)
-        else:
+        elif sql.index_exists(model._cr, indexname):
             _schema.info("Keep unexpected index %s on table %s", indexname, model._table)
```

Comment:
What does this message mean?

It's written as an instruction... are you telling the admin to keep it?

Reply:
should be written as "Keeping" but I won't argue with Odoo's wording in log messages.